### DISCUSSION TITLE
Add more CC licences + Use CC licence badges in 'Reuse' section

### DISCUSSION THREAD

**Discussion by the-solipsist** [quarto-dev/quarto-cli#6770] [Feature Requests]
### Description

1. Currently only the following four Creative Commons licences are supported:
    1. CC BY
    2. CC BY-SA
    3. CC BY-ND
    4. CC BY-NC
   
   This means that licences such as CC BY-NC-SA are not supported. It's not clear why, given that the code to support all the CC licences won't add any complexity.

2. If `licence:` metadata is CC BY-NC, and the language is en-IN, then it generates a link to https://creativecommons.org/licenses/by-nc/4.0/deed.en-in (with that URL as the text).

   It would be preferable to use [Creative Commons badges or icons](https://creativecommons.org/about/downloads/), with the appropriate URL instead. Or at the very least, to use the licence short code as the anchor text for the link.  Even better would be to generate text + machine-readable code instead of just a bare URL, but this might run into internationalization issues.

`<a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" /></a><br /><span xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Text" property="dct:title" rel="dct:type">A pantent conundrum</span> by <a xmlns:cc="http://creativecommons.org/ns#" href="https://indianexpress.com/article/opinion/columns/a-patent-conundrum/" property="cc:attributionName" rel="cc:attributionURL">Pranesh Prakash</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a>.`
rather than
`<section class="quarto-appendix-contents"><h2 class="anchored quarto-appendix-heading">Reuse</h2><div id="quarto-reuse" class="quarto-appendix-contents"><div><a rel="license" href="[https://creativecommons.org/licenses/by-nc/4.0/deed.en_in](view-source:https://creativecommons.org/licenses/by-nc/4.0/deed.en_in)">https://creativecommons.org/licenses/by-nc/4.0/deed.en_in</a></div></div></section>`

**Comment by mcanouil** [comment]
It seems to be a good/simple enhancement.

Note that PRs are always welcome and in this case, it seems straightforward for you to contribute if you feel like it: 
- https://github.com/quarto-dev/quarto-cli/blob/f87976c6dc9b0d0ea125ac58ad08860d63dae8ad/src/format/html/format-html-appendix.ts#L39-L45
- https://github.com/quarto-dev/quarto-cli/blob/f87976c6dc9b0d0ea125ac58ad08860d63dae8ad/src/resources/filters/modules/license.lua#L11-L49
- https://github.com/quarto-dev/quarto-cli/blob/f87976c6dc9b0d0ea125ac58ad08860d63dae8ad/src/resources/schema/document-metadata.yml#L49-L73

**Comment by mcanouil** [comment]
Closing in favour of
- https://github.com/quarto-dev/quarto-cli/issues/6783
